app/controllers/bulkrax/importers_controller.rb

```diff
@@ -73,7 +73,7 @@ def create
       if api_request?
         return return_json_response unless valid_create_params?
       end
-      file = params.to_unsafe_h[:importer][:parser_fields].delete(:file)
+      file = params[:importer][:parser_fields].delete(:file)
```

Comment:
@orangewolf using to_unsafe_h here means we end up with the whole tmpfile in the db, which we don't want. I know you needed this for cloud files, but is it really needed for file?

Reply:
when I tested this file was also not working - did you try uploading with a file after this change? Another approach might be to make a method that removes the item from the params, then permits that now extracted item instead of calling `to_unsafe_h`. That's probably a better way to go for both items now that I think about it.